quodlibet/po/fr.po

```diff
@@ -929,28 +908,25 @@ msgstr[1] ""
 "Voulez-vous continuer?"
 
 #: ../quodlibet/browsers/playlists/menu.py:79
-#, fuzzy, python-format
+#, python-format
 msgid "Confirm action for playlist \"%s\""
-msgstr "Confirmez la suppression de la liste de lecture"
+msgstr "Confirmez l'action pour liste de lecture\"%s\""
```

Comment:
There is a space missing between "lecture" and "\"%s\"". I would also add a "la": "[…] l'action pour **la** liste de […]"